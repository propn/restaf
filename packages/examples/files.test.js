let testFunctions = require("./testFunctions");

test("computeService: basic", async () => {
  let l = await testFunctions.filesCreate();
  expect(l).toMatchSnapshot();
});


test('computeService: basic', async () => {
	let l = await testFunctions.filesPaginate();
	expect(l).toMatchSnapshot();
});