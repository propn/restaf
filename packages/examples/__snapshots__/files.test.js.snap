// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`computeService: basic 1`] = `
Object {
  "me": "hi there",
}
`;

exports[`computeService: basic 2`] = `"done"`;
